java/.settings/JMRI PanelPro.launch

```diff
@@ -20,5 +20,5 @@
 <listEntry value="org.eclipse.debug.ui.launchGroup.run"/>
 </listAttribute>
 <booleanAttribute key="org.eclipse.jdt.launching.ATTR_USE_START_ON_FIRST_THREAD" value="true"/>
-<stringAttribute key="org.eclipse.jdt.launching.WORKING_DIRECTORY" value="${workspace_loc:/JMRI}"/>
+<stringAttribute key="org.eclipse.jdt.launching.WORKING_DIRECTORY" value="${project_loc:jmri}"/>
```

Comment:
@DanielBoudreau Is there something in or outside of Eclipse that I need to set to make this work? With this change, I get error "Base directory name can't be evaluated".